src/Cassandra/LocalTime.cs

```diff
@@ -115,7 +116,7 @@ public static LocalTime Parse(string value)
             var nanos = 0;
             if (parts.Length == 3)
             {
-                var decimalSeconds = Convert.ToDecimal(parts[2]);
+                var decimalSeconds = Convert.ToDecimal(parts[2], CultureInfo.InvariantCulture);
```

Comment:
I think we should leave it as culture dependent, I see no reason for change. We can document it in the method.

Reply:
hm... I see, good catch!